projects/core/src/checkout/facade/checkout-delivery.service.spec.ts

```diff
@@ -0,0 +1,207 @@
+import { inject, TestBed } from '@angular/core/testing';
+import { Store, StoreModule } from '@ngrx/store';
+import { CartDataService } from '@spartacus/core';
+import { Address, AddressValidation } from '../../model/address.model';
+import { Cart } from '../../model/cart.model';
+import { DeliveryMode } from '../../model/order.model';
+import * as fromCheckout from '../store/index';
+import { CheckoutDeliveryService } from './checkout-delivery.service';
+
+describe('CheckoutDeliveryService', () => {
+  let service: CheckoutDeliveryService;
+  let cartData: CartDataService;
+  let store: Store<fromCheckout.CheckoutState>;
+  const userId = 'testUserId';
+  const cart: Cart = { code: 'testCartId', guid: 'testGuid' };
+
+  const address: Address = {
+    firstName: 'John',
+    lastName: 'Doe',
+    titleCode: 'mr',
+    line1: 'Toyosaki 2 create on cart',
+    town: 'Montreal',
+    postalCode: 'L6M1P9',
+    country: { isocode: 'CA' },
+  };
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        StoreModule.forRoot({}),
+        StoreModule.forFeature('checkout', fromCheckout.getReducers()),
+      ],
+      providers: [CheckoutDeliveryService, CartDataService],
+    });
+
+    service = TestBed.get(CheckoutDeliveryService);
+    cartData = TestBed.get(CartDataService);
+    store = TestBed.get(Store);
+
+    spyOn(store, 'dispatch').and.callThrough();
+  });
+
+  it('should CheckoutDeliveryService is injected', inject(
```

Comment:
`should CheckoutDeliveryService is injected` - here i think is something wrong :) maybe `to be injected`?